dsc/TOC.md

```diff
@@ -54,6 +54,10 @@
 ## [Using a DSC report server](reportServer.md)
 ## [Pull server best practices](secureServer.md)
 
+# [DSC examples](dscExamples.md)
+## [Building a CI-CD pipeline with DSC, Pester, and Visual Studio Team Services](dscCiCd.md)
```

Comment:
Would it be better to spell out the CI-CD acronym and remove "...with DSC, Pester, and Visual studio..." from the title of this article?